akka23-common/src/main/scala/com/typesafe/conductr/lib/akka/ConnectionHandler.scala

```diff
@@ -147,6 +135,10 @@ class ConnectionHandler extends AbstractConnectionHandler {
     requestF.map { request =>
       request.addHeader(`User-Agent`(UserAgent))
       request.addHeader(Host(url.getHost, url.getPort))
+      payload.getRequestHeaders.foldLeft(request) { (request, entry) =>
+        val (header, headerValue) = entry
+        request.addHeader(RawHeader(header, headerValue))
+      }
```

Comment:
I have now taken care of Java, Scala, and Play. There doesn't seem to be any `ConnectionHandler` for Lagom.